### ISSUE TITLE
Support local file uri

### ISSUE THREAD

**Issue by huhuanming** [DylanVann/react-native-fast-image#23]
On ios, local file uri is `/xxx/ooo/document/a.jpg`.
And on Android, the uri is `file://xxx/ooo/document/a.jpg`.

Could I add something to support it ?

**Comment by DylanVann** [comment]
Hm.. potentially. Maybe if the url doesn't include http we fallback to React Native's `Image`. Right now it checks for `source.uri`, if it's there it tries to render it with the native libraries.

**Comment by huhuanming** [comment]
I found it already support local uri after testing.

But i found another bug.

let me think how to describe it...

**Comment by huhuanming** [comment]
if you set props to React Component like this.

```
<Component
     a={1}
     b={1}
     c={1}
/>
```

In the native code, the `a`,  `b`, `c` cannot be assigned value at the same time.

Maybe `a` is first,  or not.

So, I fixed it by adding `reloadImage` function like the following link.

`https://github.com/huhuanming/react-native-fast-image/blob/master/ios/FastImage/FFFastImageView.m`

To proactively satisfy the our project requirements,  I changed other code, do not mind.
